irods/test/login_auth_test.py

```diff
@@ -511,20 +512,18 @@ def test_ssl_with_server_verify_set_to_none_281(self):
         with helpers.file_backed_up(env_file):
             with open(env_file) as env_file_handle:
                 env = json.load( env_file_handle )
-            env.update({ "irods_client_server_negotiation": "request_server_negotiation",
-                         "irods_client_server_policy": "CS_NEG_REQUIRE",
-                         "irods_ssl_ca_certificate_file": "/path/to/some/file.crt",  # does not need to exist
-                         "irods_ssl_verify_server": "none",
-                         "irods_encryption_key_size": 32,
-                         "irods_encryption_salt_size": 8,
-                         "irods_encryption_num_hash_rounds": 16,
-                         "irods_encryption_algorithm": "AES-256-CBC" })
+            my_ssl_directory = os.path.expanduser("~/some")
```

Comment:
Is it still the case that this directory need not exist?

Reply:
Altering this to be a temporary directory, which it should have been to begin with.